File: I'm everywhere.js
function i(word) {
  const vowel_c = (word.match(/[aiueo]/gi)|| []).length;
  if(word!=='' && word[0]!=="I" && vowel_c < word.length-vowel_c && word[0].toUpperCase()===word[0]){
    return "i"+word[0].toUpperCase()+word.slice(1,);
  }
  return "Invalid word"
}

File: I'm everywhere.py
def i(word):
    vowel_c = sum([1 for s in word if s.lower() in ['a','i','u','e','o']])
    if len(word)>0 and word[0].lower()!="i" and vowel_c < len(word)-vowel_c and word[0].isupper():
        return "i"+word[0].upper()+word[1:]
    return "Invalid word"
    
